### PR TITLE
Create understand-and-accelerate-first-time-open-source-contributions.md

### PR DESCRIPTION
**Pull request by logankilpatrick**
## Reasons for making this change

Submitting new proposal

### Related issues

N/A

### CODE REVIEW

**Comment by logankilpatrick** [comment]
Hey all! Looking forward to getting feedback on this.

**Comment by epicfaace** [comment]
Interesting, thanks for sharing!!

How would the NLP proposal differ from / potentially work with https://github.blog/2020-01-22-how-we-built-good-first-issues/?

**Comment by coni2k** [comment]
It's entirely a different approach but let me share this old article; maybe it will give new ideas:
https://forcrowd.medium.com/first-mission-contributors-page-df24e6e70705

In short, is it possible to create a fixed set of tasks/challenges for the new contributors to understand the codebase, similar to a video game tutorial?

Recently I found the First Contributions repository, which looks highly successful:
https://github.com/firstcontributions/first-contributions

In other words, extend this approach to create a standard checklist for different project types (web, API, library, database) that the maintainers can apply to their repositories.

My two cents.

**Comment by logankilpatrick** [comment]
From my perspective as a contributor to open source, the existing model does not do a good job, if you look through the issues it is tagging, they do not have the context for a beginner to jump in and contribute. In this proposal, I'm hoping to go a few steps further than what GitHub does and really identify issues where someone without a deep background in a specific project can jump in and contribute. I'll re-word the proposal to reflect this! Also, this discussion has helped a ton, so thanks for all the feedback so far!

**Comment by coni2k** [comment]
> It's entirely a different approach

@epicfaace I wrote that comment, and I was referring to my article.

I wanted to share that article since it's also related to improving the new contributors' experience. I thought it might give ideas to someone else. But my focus was completely different and not connected with @logankilpatrick's proposal.

So, feel free to ignore it :)

**Comment by epicfaace** [comment]
> I'll re-word the proposal to reflect this! Also, this discussion has helped a ton, so thanks for all the feedback so far!

Great! Once you re-word this proposal I think we can get this merged.

**Comment by epicfaace** [comment]
Hi @logankilpatrick ! We've clarified the process for proposals to be submitted and then discussed -- so once you make those changes, we can merge it and create a Discussion thread for it! See https://github.com/PlaintextGroup/oss-virtual-incubator#3-next-steps-after-submitting.

**Review by epicfaace — APPROVED** [review]
I'm good to merge this for now! Feel free to add any changes in a follow-up.
